src/api/app/views/webui/user/notifications.html.erb

```diff
@@ -3,19 +3,20 @@
 
 <h2><%= @pagetitle %></h2>
 
-<% gs = User.current.groups_users %>
-
 <%= form_tag({ :action => :update_notifications }, id: 'notification_form') do %>
 
-    <% unless gs.empty? %>
+    <% unless @groups.empty? %>
         <h3>Get mails if in group ...</h3>
         <ul style="list-style-type:none;">
-          <% gs.each do |gu| %>
-              <li><%= check_box_tag gu.group, '1', gu.email %><%= label_tag gu.group.title %></li>
+          <% @groups.each do |gu| %>
+              <li>
+                <%= check_box_tag gu.group, '1', gu.email %>
+                <%= label_tag gu.group.title %>
+              </li>
           <% end %>
         </ul>
     <% end %>
 
-    <%= render partial: 'webui/notifications/roles' %>
+    <%= render partial: 'webui/notifications/roles', locals: { user: User.current } %>
```

Comment:
I prefer to pass User.current as a @user variable from the controller action. This view doesn't have to know what to pass to the partial... from my POV it belongs to the controller here.